idaes/generic_models/properties/core/state_definitions/FPhx.py

```diff
@@ -39,6 +44,23 @@ def define_state(b):
     # TODO: should have some checking to make sure developers implement this properly
     b.always_flash = True
 
+    # Check that only necessary state_bounds are defined
+    expected_keys = ["flow_mol", "enth_mol", "temperature", "pressure"]
```

Comment:
Question: Why is `temperature` an expected key when the state definition is `FPhx`?

Reply:
Temperature is there to allow users to provide bounds for T as they are generally better known than bounds on enthalpy. These are also very helpful (if not necessary) for initializing the flash calculations.